seeders/reviewsSeeder.ts

```diff
@@ -40,16 +40,19 @@ const seedReviews = async (prisma: PrismaClient): Promise<void> => {
       ratingSum += rating;
     }
 
-    const rating = (await ratingSum) / dataQuantity;
-
-    await prisma.restaurants.update({
-      where: {
-        id_restaurant: id_restaurant,
-      },
-      data: {
-        rating,
-      },
-    });
+    setTimeout(async () => {
+      const rating = ratingSum / dataQuantity;
+      await prisma.restaurants.update({
+        where: {
+          id_restaurant: id_restaurant,
+        },
+        data: {
+          rating,
+        },
+      });
+
+      console.log("Finished");
+    }, 5000);
   }
```

Comment:
I propose this refactor:
```js
function delay(resolve) {
  return new Promise((resolve, reject) => {
    setTimeout(() => {
      resolve();
    }, 5_000);
  });
}

 const rating = ratingSum / dataQuantity;
 const updateRestaurants = prisma.restaurants.update({
    where: {
       id_restaurant: id_restaurant,
     },
    data: {
      rating,
    },
});

delay(updateRestaurants);
```

Reply:
I tried to apply the proposed refactoring, but an error is generated with the rating variable and the execution is aborted